126 spacing between elements

vsheo opened the pull request:
## What does this change?

Resolves issue #126 

We all worked separately on our own components, which resulted in slightly different margins and padding everywhere.  
In this branch, I created custom properties for spacing and used them to apply consistent whitespace between elements

## Images
Before:
<img width="2850" height="8286" alt="designingforrecognition netlify app_" src="https://github.com/user-attachments/assets/7a9a02db-25e0-48d4-8bdb-edbb2637c43b" />

After:
<img width="2850" height="6182" alt="localhost_5173_" src="https://github.com/user-attachments/assets/c1a2bc49-762c-40c3-8553-8ef44101995b" />



## How to review
- Review the website across multiple pages and screen sizes (mobile and desktop).
- Focus specifically on spacing and whitespace consistency between elements.
- Check whether the spacing feels consistent, especially between:
  - section titles
  - filters
  - project cards
  - links in the footer

Compare different pages and components to see if the same types of elements use the same spacing.

Look for any places where:
- spacing feels too tight or too loose
- margins or paddings feel inconsistent with the rest of the site
- spacing could be improved for better hierarchy or readability

Please let me know:
- If you notice any inconsistent spacing
- If certain areas would benefit from more or less whitespace
- If the current spacing scale feels clear and visually balanced
- If there are components that should use a different spacing value

vsheo committed:
fix: footer feedback restored

vsheo committed:
fix: filter component spacing #126

vsheo committed:
fix: project container spacing #126

vsheo committed:
fix: about us spacing #126

vsheo committed:
fix: footer spacing #126

vsheo committed:
fix: line height #157

vsheo committed:
fix: 2em after paragraph #106

vsheo committed:
fix: letter and word spacing #106

vsheo committed:
feat: modular scale for font-sizes #108

Abeeryu commented:
@vsheo De spacing met custom properties is goed opgezet en de spacing-schaal (--spacing-l, --spacing-m, --spacing-s) is duidelijk en logisch. De ruimte tussen secties, is ook duidelijke scheiding tussen onderdelen.

Titels hebben consistente ruimte door bijvoorbeeld margin-bottom: var(--spacing-m), wat de hiërarchie versterkt.

Je gebruikt geen vaste waarden zoals: 0.5rem 0 1rem 0 en margin: 2em. Misschien kan je dit vervangen door var(--spacing-xs) of var(--spacing-s) voor meer consistentie.

De spacing in de footer is goed, maar de ruimte tussen links wisselt nog (bijvoorbeeld gap: 0.5rem versus gap: 1em). Hier kan je 1 spacing variabele gebruiken.

FatimaZelay commented:
Ik heb hier wat punten die je eventueel zou kunnen verbeteren:

- Verander de filter buttons in een ander kleur om duidelijk aan te geven aan de gebruiker dat het om een ander soort button gaat
- Probeer misschien ook dezelfde breedte aan de buttons te geven zodat het consistent uitziet.

Hieronder zie je wat ik bedoel.

<img width="696" height="361" alt="Screenshot 2026-01-09 at 15 06 24" src="https://github.com/user-attachments/assets/05e568e8-9bcf-4b17-9a7c-c05f33ff0dce" />

Karima002 commented:
Goed bezig!

- De e in Experiment mist nog een hoofdletter.
- De padding op de buttons lijkt niet helemaal gelijk te zijn, kijk hier nog even naar!
<img width="326" height="739" alt="Scherm­afbeelding 2026-01-09 om 15 04 00" src="https://github.com/user-attachments/assets/f4279d96-f185-4472-9d06-13d2495f8f66" />

- Op een scherm kleiner dan rond de 400px vervaagt de kleur van de menu in de kleur van de buttons. Op een scherm groter dan 400 px is er wel een padding te zien. Wellicht zou je dit ook kunnen toepassen op schermen kleiner dan 400px.


<img width="352" height="749" alt="Scherm­afbeelding 2026-01-09 om 15 11 07" src="https://github.com/user-attachments/assets/62578747-fab1-4f5f-82b3-632d4c5959df" />

vsheo committed:
fix: text wrap balance on titles #107

vsheo committed:
Fix: filter options & buttons #126

vsheo committed:
fix: padding between buttons & filters #126

vsheo committed:
feat: menu extra closing options #106

vsheo commented:
@Abeeryu , @FatimaZelay and @Karima002  
Thank you for the feedback, which I have now addressed. These are the changes that were made:


I gave the activate filter and “Reset filters” buttons a different color to make it clear that these are distinct from the other buttons. I also set all filter buttons to the same width to create a more consistent and uniform appearance.

When the hamburger menu is open, it overlaps the filters, which previously meant the user could only close it by clicking the close icon. I added a dark overlay to the visible part of the page, clicking this overlay now also closes the menu.
In addition, the menu now closes automatically when the user scrolls.
Before:

[2026-01-13 09-46-29.webm](https://github.com/user-attachments/assets/6271e070-f3ec-44e0-8115-ac0c46d1df6c)

After:

[2026-01-13 16-01-14.webm](https://github.com/user-attachments/assets/ba71711b-85b1-4432-a04e-6ebd55bb6915)

Mikiyas-hs commented:
<img width="348" height="625" alt="image" src="https://github.com/user-attachments/assets/129c8b0c-7784-494f-b5e1-c13fab9ea637" /> and this is how it looks on desktop: 
<img width="1277" height="752" alt="image" src="https://github.com/user-attachments/assets/81f3973b-5535-463d-81d0-df6008502cd5" />


The padding here makes the sponsor-carousel look smaller thats because of the full width it takes of the footer

vsheo committed:
Merge pull request #161 from fdnd-agency/107-text-alignment

107 text alignment

vsheo committed:
fix: footer padding #156

vsheo committed:
fix: footer code cleanup #156

vsheo commented:
I have fixed these issues and will merge this into the dev branch now.
